pkg/operator/controller.go

```diff
@@ -1,481 +1,302 @@
-			c.workqueue.AddRateLimited(wqKey{kind: wqKindProfile, namespace: ntoconfig.OperatorNamespace(), name: nodeName})
-		}
-		return nil
-
-	case key.kind == wqKindNode:
-		klog.V(2).Infof("sync(): Node %s", key.name)
-
-		change, err := c.pc.nodeChangeHandler(key.name)
-		if err != nil {
-			if errors.IsNotFound(err) {
-				// Do not leave any leftover profiles after node deletions
-				klog.V(2).Infof("sync(): deleting Profile %s", key.name)
-				err = c.clients.Tuned.TunedV1().Profiles(ntoconfig.OperatorNamespace()).Delete(context.TODO(), key.name, metav1.DeleteOptions{})
-				if err != nil && !errors.IsNotFound(err) {
-					return fmt.Errorf("failed to delete Profile %s: %v", key.name, err)
-				}
-				klog.Infof("deleted Profile %s", key.name)
-				return nil
-			}
-			return fmt.Errorf("failed to process Node %s change: %v", key.name, err)
-		}
-		if change {
-			// We need to update Profile associated with the Node
-			klog.V(2).Infof("sync(): Node %s label(s) changed", key.name)
-			// Trigger a Profile update
-			c.workqueue.AddRateLimited(wqKey{kind: wqKindProfile, namespace: ntoconfig.OperatorNamespace(), name: key.name})
-		}
-		return nil
-
-	case key.kind == wqKindMachineConfigPool:
-		klog.V(2).Infof("sync(): MachineConfigPool %s", key.name)
-
-		// MachineConfigPool changes may mean the operator-created MachineConfigs are no longer needed.
-		// Note this is not only direct MachineConfigPool changes, but also indirect changes, such as
-		// removing labels from nodes so that they no longer are part of a given MCP.
-		err = c.pruneMachineConfigs()
-		if err != nil {
-			return err
-		}
-
-		// MachineConfigPool changes can affect all nodes and MCP is where cluster admins
-		// will adjust the operator behavior when using the MachineConfig functionality.
-		// Nodes can become part of the pool or they can lose the pool membership.
-		// Trigger profile calculations/updates for all Tuned Profiles in the cluster.
-		err = c.enqueueProfileUpdates()
-		if err != nil {
-			return err
-		}
-
-		return nil
-
-	case key.kind == wqKindDaemonSet, key.kind == wqKindClusterOperator:
-		klog.V(2).Infof("sync(): DaemonSet/OperatorStatus")
-
-		err = c.syncDaemonSet(cr)
-		if err != nil {
-			return fmt.Errorf("failed to sync DaemonSet: %v", err)
-		}
-		err = c.syncOperatorStatus(cr, nil)
-		if err != nil {
-			return fmt.Errorf("failed to sync OperatorStatus: %v", err)
-		}
-		return nil
-
-	case key.kind == wqKindProfile:
-		klog.V(2).Infof("sync(): Profile %s", key.name)
-
-		// Check if node labels are already cached.  If not, ignore this event.
-		// Profile update is triggered later on once node labels are cached on
-		// the node event.
-		if c.pc.state.nodeLabels[key.name] == nil {
-			return nil
-		}
-
-		err = c.syncProfile(cr, key.name)
-		if err != nil {
-			return fmt.Errorf("failed to sync Profile %s: %v", key.name, err)
-		}
-		return nil
-
-	default:
-	}
-
 	// Tuned CR changed and the operator components need to be managed.
```

Comment:
probably we should update the comment to reflect what really happens

Reply:
I would just remove it in our case